touchforms/formplayer/api.py

```diff
@@ -90,10 +93,13 @@ def get_touchforms_dict(self):
                 ("instance-content", self.instance_content),
                 ("preloader-data", self.preloader_data),
                 ("session-data", self.session_data),
-                ("lang", self.language))
+                ("lang", self.language),
+                ("form-url", self.form_path))
         
         # only include anything with a value, or touchforms gets mad
-        return dict(filter(lambda x: x[1], vals))
+        ret = dict(filter(lambda x: x[1], vals))
+        ret.update(self.session_data)
```

Comment:
i just noticed that this is going to duplicate every item in session_data at the root level of the dict. is this necessary, or can we be more selective about the items that formplayer needs instead of adding everything?